src/core/core.controller.js

```diff
@@ -111,12 +111,15 @@ function initConfig(config) {
 
 	const scaleConfig = mergeScaleConfig(config, config.options);
 
-	config.options = mergeConfig(
+	const options = config.options = mergeConfig(
 		defaults,
 		defaults[config.type],
 		config.options || {});
 
-	config.options.scales = scaleConfig;
+	options.scales = scaleConfig;
+
+	options.title = (options.title !== false) && merge({}, [defaults.plugins.title, options.title]);
```

Comment:
Maybe we should just make this `options.title` instead of `options.title !== false` to handle the case where it's `null` or `undefined`

Reply:
maybe. this is how its been before though, setting explicitly to false to disable a plugin. undefined/null does not disable the plugin, but uses default options instead.

Reply:
Yeah, makes sense. Let's leave it as is